### DISCUSSION TITLE
Add the option to average data from sources

### DISCUSSION THREAD

**Discussion by kylagit** [breezy-weather/breezy-weather#1180] [Ideas]
### Describe your idea

The Breezy weather app should have an option to average data from all weather sources that have data for your location.

So if your location is covered by two sources, and one source claims it is 16°C outside, while the other source claims it is 18°C outside, then 17°C should be displayed by the app.

This helps account for differences in the data collected by weather data sources.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
This is not how models work.

If one model tells you it's 20 °C because it thinks the thunderstorm is not going through your area with 10 % probability, and the other thinks it's going to be 10 °C with 90% probability... Then showing 15 °C with 45 % probability would make no sense, because this scenario exists in none of these models and it would just be garbage data. Of course, it's a simplified example but you get the idea.

Have a look at Ensemble models if you are into forecast scenarios.
